src/cortex_cli/cortex_cli.py

```diff
@@ -218,33 +311,73 @@ def status(config_file, verbose) -> None:
-    config = read_json(config_file)
+    config = _validate_config_file(config_file)
     tokens_file = config['tokens_file']
     if not Path(tokens_file).is_file():
         click.echo('Not logged in. Use "cortex auth login" to login.')
         return
 
-    tokens_data = read_json(tokens_file)
+    tokens_data = _validate_tokens_file(tokens_file)
 
     click.echo(f'Tokens file: {tokens_file}')
     if 'pid' not in tokens_data:
-        click.echo("Tokens file doesn't contain PID. Probably, 'cortex auth login' was launched with '--no-daemon'\n")
+        click.echo("Tokens file doesn't contain PID. Probably, 'cortex auth login' was launched with '--no-refresh'\n")
 
     click.echo(f"Last refresh: {tokens_data['timestamp']}")
     seconds_at = time_left_seconds(tokens_data['access_token'])
-    time_left_at = str(datetime.timedelta(seconds=seconds_at))
+    time_left_at = str(timedelta(seconds=seconds_at))
     click.echo(f'Time left on access token (hh:mm:ss): {time_left_at}')
     seconds_rt = time_left_seconds(tokens_data['refresh_token'])
-    time_left_rt = str(datetime.timedelta(seconds=seconds_rt))
+    time_left_rt = str(timedelta(seconds=seconds_rt))
     click.echo(f'Time left on refresh token (hh:mm:ss): {time_left_rt}')
 
-    active_pid = check_daemon(tokens_file)
+    active_pid = check_token_manager(tokens_file)
     if active_pid:
         click.echo(f'Token manager: {click.style("RUNNING", fg="green")} (PID {active_pid})')
     else:
         click.echo(f'Token manager: {click.style("NOT RUNNING", fg="red")}')
 
 
+def _validate_cortex_cli_auth_login(no_daemon, no_refresh, config_file) -> dict:
+    """Checks if provided combination of auth login options is valid:
+       - no_daemon and no_refresh are mutually exclusive
+       - daemon mode should not be requested on Windows
+       - config file should pass validation
+
+    Args:
+        config_file (str): --config-file option value
+        no_daemon (bool): --no-daemon option value
+        no_refresh (bool): --no-refresh option value
+    Raises:
+        click.BadOptionUsage: if both mutually exclusive --no-daemon and --no-refresh are set
+        click.UsageError: if daemon is requested on Windows
+    Returns:
+        dict: config dict loaded from config_file
+    """
+
+    # --no-refresh and --no-daemon are mutually exclusive
+    if no_refresh and no_daemon:
+        raise click.BadOptionUsage(
+            '--no-refresh',
+            "Cannot request a non-daemonic (foreground) token manager when using '--no-refresh'.")
+
+    # daemonizing is unavailable on Windows
+    if platform.system().lower().startswith('win') and not no_refresh and not no_daemon:
+        raise click.UsageError(
+            "Daemonizing is not yet possible on Windows. Please, use '--no-daemon' or '--no-refresh' flag.")
+
+    # config file, even the default one, should exist
+    if not Path(config_file).is_file():
+        raise click.BadParameter(
```

Comment:
Missing from the "Raises" section of the docstring.

Reply:
Fixed.